GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -296,6 +297,52 @@ public GVRMesh createQuad(float width, float height) {
     }
 
     /**
+     * Generates default color for missing textures in the modelling file.
+     *
+     */
+    private static final int WIDTH = 50;
+    private static final int HEIGHT = 50;
+    private static final int STRIDE = 64;    // must be >= WIDTH
+
+    private static int[] createColors() 
+    {
+        int[] colors = new int[STRIDE * HEIGHT];
+        for (int y = 0; y < HEIGHT; y++)
+        {
+            for (int x = 0; x < WIDTH; x++)
+            {
+                int r = x * 255 / (WIDTH - 1);
+                int g = y * 255 / (HEIGHT - 1);
+                int b = 255 - Math.min(r, g);
+                int a = Math.max(r, g);
+                colors[y * STRIDE + x] = (a << 24) | (r << 16) | (g << 8) | b;
+            }
+        }
+        return colors;
+    }   
+
+    /**
+     * Loads the complete scene from the imported 3D modelling file.
+     * 
+     * @param fileName
+     *            The name of a 3D model file, relative to the assets directory.
+     *            The assets directory may contain an arbitrarily complex tree
+     *            of subdirectories; the file name can specify any location in
+     *            or under the assets directory.
+     */
+    public void loadScene(String fileName)
+    {
+        GVRAssimpImporter assimpImporter = GVRImporter.readFileFromAssets(this, fileName);
+        //Default bitmap for objects whose texture are not present or there is some texture error
+        int[] mColors = createColors();
+        int[] colors = mColors;
+        Bitmap defaultBitmap = Bitmap.createBitmap(colors, 0, STRIDE, WIDTH, HEIGHT, Bitmap.Config.RGB_565);
```

Comment:
Please use an `res/drawable-xxxhdpi` bitmap for this `defaultBitmap` instead of embedding it in code. This will let use replace it with a designer-supplied graphic if necessary.